web/views.py

```
@@ -4643,6 +4652,145 @@
 
     return render(request, "teams/delete_confirm.html", {"goal": goal})
 
+@login_required
+def virtual_classroom_list(request):
```

Ruff (0.11.9)

Missing return type annotation for public function `virtual_classroom_list`

(ANN201)